backend/novion.py

```
@@ -66,53 +93,201 @@ def supervisor_node(state: State) -> Command[Literal["pharmacist", "researcher",
+"""
+
+# Create the base pharmacist agent
+pharmacist_base_agent = create_react_agent(
+    llm, 
+    tools=[get_drug_use_cases, search_drugs_for_condition],
+    prompt=pharmacist_system_prompt
+)
+
+# Enhance with MCP tools (particularly useful for medication data)
+pharmacist_agent = enhance_agent_with_mcp(
+    pharamcist_base_agent,
```

Ruff (0.8.2)

Undefined name `pharamcist_base_agent`

(F821)